package com.FileExceptionHandling.testing;

public class ArithemeticException {

	public static void main(String[] args) {

		int a = 10;
		int b = 0;

		/*try {
			System.out.println(a / b);
		} catch (ArithmeticException e) {
			System.out.println("Can't divide number by 0");
			System.out.println("the error message is " + e);
		}

		System.out.println("This is the end of the statement");
*/
		
		
		
	}

}
